webserver/templates/user/scraper.js

```diff
@@ -149,6 +152,11 @@ function reportScrobbles(struct) {
     xhr.setRequestHeader("Content-Type", "application/json;charset=UTF-8");
     xhr.onload = function(content) {
         numCompleted++;
+        if (numCompleted >= numberOfPages) {
+            updateMessage("<i class='fa fa-check'></i> Import finished<br><span style='font-size:8pt'>Thank you for using ListenBrainz</span>");
+        } else {
+            updateMessage("<i class='fa fa-cog fa-spin'></i> Sending page " + numCompleted + " of " + numberOfPages + " to ListenBrainz<br><span style='font-size:8pt'>Please don't navigate while this is running</span>");
+        }
         console.log("successfully reported page");
```

Comment:
For next steps it might be nice to show a close button that allows the user to return to last.fm